src/Ethereum.sol

```diff
@@ -173,9 +176,11 @@ library Ethereum {
     address internal constant SUPPLY_LOGIC      = 0x46256841e36b7557BB8e4c706beD38b17A9EB2c1;
 
     /******************************************************************************************************************/
-    /*** L2 Forwarders                                                                                              ***/
+    /*** Cross-Domain Addresses                                                                                     ***/
     /******************************************************************************************************************/
 
+    address internal constant CCTP_MESSENGER = 0xBd3fa81B58Ba92a82136038B25aDec7066af3155;
```

Comment:
This should probably be labelled `CCTP_TOKEN_MESSENGER` to be more clear.

Reply:
Updated